### DISCUSSION TITLE
Switch port

### DISCUSSION THREAD

**Discussion by mc115x** [HarbourMasters/2ship2harkinian#656] [Ideas]
Will this at some point receive a switch port like the of ship of harkinian?

**Comment by caller0** [comment]
Hopefully, it will (I guess). Can't wait for it.

**Comment by hobbo91** [comment]
I would also very much like a port for the Nintendo switch. For me the OLED Switch couldn't be any more perfect for SoH. As much as OOT is probably my #1 of all time it would be awesome to play a native port of MM as well on the Switch.

Are there any plans for this?

**Comment by disavow-commits** [comment]
https://github.com/YoshiCrystal9/2ship2harkinian-Switch

This port works well.  I hope it gets included into this main github.
